docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -4,6 +4,29 @@ Coding style
 
 Please follow these coding standards when writing code for inclusion in Django.
 
+
```

Comment:
`.. _coding-style-pre-commit: `

Reply:
At the risk of "treading on each other toes" I'll go do my day job now. I'm very happy to pick these up later but in the meantime gives either yourself/Carlton the opportunity to make these without me getting in the way.